6 authorization edit

VisualVortex opened the pull request:
Внес следующие изменения:
 Поменять title страницы
 Центрировать блок
 Попробовал припилить Адаптивность
 Добавил фавиконку
 Переименовать с регистрация на авторизация

VisualVortex committed:
#6 - Поменял title страницы авторизации

VisualVortex committed:
Центрировал, +адаптация, подогнал стиль, заменил пару слов и опечаток.

VisualVortex committed:
#6 - Центрировал, +адаптация, подогнал стиль, заменил пару слов и опечаток.

VisualVortex committed:
#6 - Добавил favicon

VisualVortex committed:
#6 - Добавил favicon файл

VisualVortex committed:
close #6 исправил немного адаптацию

EAndreyF commented:
Вроде ок. Жень, посмотри

VisualVortex committed:
поменял все табы, которые нашел в проекте на четыре пробела

VisualVortex commented:
Нет, Андрей - не ОК :) Страшный баг я создал.
Никогда не работал с цсс препроцессорами  и до шри даже не слышал.  
Я вносил изменения прямиком в CSS файл, что кажется нельзя делать когда используем препроцессор ЦСС Стайлус. Я конечно не уверен, но кажется он перекомпилирует цсс файл, который я писал.
Сейчас изучу стайлус и изменю код :)

EAndreyF commented:
Добавь тогда index.css в gitignore, и удали из репозитория, чтобы таких проблем больше не было

VisualVortex committed:
#6 - переписал адаптацию в файл стайлуса, добавил в гитИгнор app/loginPublic/index.css

VisualVortex committed:
#6 - удалил файл app/loginPublic/index.css из Репо

#6 - удалил файл app/loginPublic/index.css из Репозитория на ветке 6_authorization-edit. В прошлом коммите добавил этот файл в гитИгнор.

VisualVortex committed:
#6 - в стилусах везде добавил : и убрал ; а так же чуть поправил адаптацию

VisualVortex commented:
Ребята, можете посмотреть. Стайлус осилил и все переписал. Проверяйте https://github.com/ya-shred/production/pull/13

Drekrosh commented:
Не годится. Я сделаю так,  как надо по дезу, зачем запихнули все по центру не понятно. Если и надо было центрировать , то только по вертикали. Короче , я против. Я сделаю под 2560 разрешения и до 440px

VisualVortex commented:
Потому что полное центрирование - удобно и оптимально по юзабилити. На большинстве крупных сайтах так и поступают. 
А при высоком разрешение если авторизация справа - будет удобна только если ты её безбожно увеличишь.
ОК ты сделаешь для 2560, а чел откроет на более широком экране (например 4к или прсото длинном), или у него масштаб сильно занижен. То ему придется смотреть сильно влево.

Rahnar commented:
Мы не все , и у нас свое виденье на  этот счет, то как у большинства меня совершенно мало интересует.

Commit:
Авторизация
